### PR TITLE
Styling 2

### PR DESCRIPTION
**Pull request by dhruv-karan**
Tpo, Principal, Rti, Commiittees and faculty pages added

### CODE REVIEW

**Comment by s-r-aman** [comment]
# Problems with the code @dhruv-karan 
1. Take back the changes of setupTests.js, jest.config.js, package.json, moment mock.
2. Take back the administrator navbar tab i.e. `src/components/header/NavBar/Adminstration/Dean.js` ,`src/components/header/NavBar/Adminstration/Administration.js`  and changes from `src/components/header/NavBar/NavBar.js`.
3. Take back `src/pages/publicPages/StudentRegistrationPage.js`.
4. RTIData file should be called something else, 'RTI' wil also do fine.
5. Do not repeat code keep all the data in one file and map that thing.
6. Change the Facility to Faculty in all positions where you have written i.e. variable names and file names and imports and exports.
7. Make a different json file for faculty data and make components pure.
8. Use ES6 imports syntax in faculty data and in all the places where you have used.
9. Use object loop here for of or object,keys etc.
10. Move all pages from static pages directory to public pages else explain the condition on which you are deciding to put these files in separate directory.
11. Take all tests related file back we will discuss how to do them.
12. ESLint giving errors in file - 
    - `src/components/static/Principal/PrincipalData.js`.
    - `.src/components/static/Principal/PrincipalSheet.js`.
    - `src/components/static/Rti/RtiData.js`.
    - `src/components/static/Rti/RtiItem.js`.
    - `src/components/static/Rti/RtiTabel.js`.
    - `src/components/static/Tpo/TpoData.js`.
    - `src/components/static/Tpo/TpoTabel.js`.
    - `src/components/static/Tpo/TpoTabelItem.js`.
    - `src/components/static/committees/Committee.js`.
    - `src/components/static/committees/CommitteesItem.js`.
    - `src/components/static/committees/CommitteesTabel.js`.
    - `src/components/static/facility/FacilitySheet.js`.
    - `src/components/static/facility/facilityLines.js`.
    - `src/pages/staticPages/PrincipalPage.js`.
    - `src/pages/staticPages/RtiPage.js`.

File: src/components/static/Faculty/Faculty.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { H2ResAuto, Container, Wrapper } from 'theme/Components';
+import { Upper, Lower } from './FacultySheet';
+import { upper, lower } from './FacultyData';
+
+const UpperData = () => (
+  <Container>
+    <H2ResAuto>FACULTY</H2ResAuto>
```
Comment (s-r-aman) [inline src/components/static/Faculty/Faculty.js]: Why is everything in caps? Make it in the proper casing.

File: src/components/static/Faculty/FacultyLines.js [hunk 2]
```diff
@@ -52,10 +52,10 @@ const Lowerlines = ({ departName, post, name, link, facility }) => (
     <A padding="0px" href={link} target="_blank">
       View
     </A>
-    <p>
+    <div>
       <Head> Faculty </Head>
-      {facility.map((item, i=0) => ( <Name key={i++}> {item} </Name> ))}
-    </p>
+      {facility.map((item, i = 0) => <Name key={i++}> {item} </Name>)}
```
Comment (s-r-aman) [inline src/components/static/Faculty/FacultyLines.js]: What are you trying to do here with i?

**Comment by dhruv-karan** [comment]
Y r u not merging this request .

**Comment by s-r-aman** [comment]
Did not got time to do so. I will try to get back a this today.

File: src/components/static/Tpo/Tpo.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  <Container>
+    <Wrap>
+      <Teachers teacherInfo={tpoTeacherCodinater} />
+    </Wrap>
+  </Container>
+);
+
+const TopData = () => (
+  <Container>
+    <H2ResAuto> TRAINING AND PLACEMENT CELL </H2ResAuto>
```
[the remark this answers is not in the document]
Comment (dhruv-karan) [reply src/components/static/Tpo/Tpo.js]: done

File: src/components/static/Tpo/Tpo.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    <br />
+  </Container>
+);
+
+module.exports = {
+  TeacherData,
+  TopData,
+  CodinaterData,
+  Placement,
+};
```
[the remark this answers is not in the document]
Comment (dhruv-karan) [reply src/components/static/Tpo/Tpo.js]: done

File: src/components/static/Tpo/TpoData.json [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+          },
+      
+          {
+            "year": "Placement 2011-12",
+            "link": "/placement_2011_12"
+          }
+    
+        ]
+
+  }
```
[the remark this answers is not in the document]
Comment (dhruv-karan) [reply src/components/static/Tpo/TpoData.json]: done

File: src/components/static/Tpo/TpoTabel.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+      {codinaterInfo.map((item) => (<Cod key={item.name} {...item} />))}
+    </tbody>
+  </StyledTable>
+);
+
+module.exports = {
+  Teachers,
+  Tops,
+  Codinaters,
+};
```
[the remark this answers is not in the document]
Comment (dhruv-karan) [reply src/components/static/Tpo/TpoTabel.js]: done

File: src/components/static/Tpo/TpoTabelItem.js [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+const Top = ({ name, contactNumber, email }) => (
+  <Contain>
+    <Name> {name} </Name>
+    <P>Training & Placement Officer </P>
+    <P> Contact Us: {contactNumber} </P>
+    <P> Email: {email} </P>
+    <br />
+    <br />
+    <P1> Training, Placement and Students' Personality Development Committee </P1>
```
[the remark this answers is not in the document]
Comment (dhruv-karan) [reply src/components/static/Tpo/TpoTabelItem.js]: done

File: src/components/static/Tpo/TpoTabelItem.js [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+    <TD title>{name}</TD>
+    <TD> {year}</TD>
+  </TR>
+);
+
+module.exports = {
+  Teacher,
+  Top,
+  Cod,
+};
```
[the remark this answers is not in the document]
Comment (dhruv-karan) [reply src/components/static/Tpo/TpoTabelItem.js]: done

File: src/components/static/Rti/RtiData.json [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+        "infoOfficer": "First Appellate Authority"
+    },
+    {
+        "name": "Mr. S.N.Sinha, (Accounts Officer)",
+        "infoOfficer": "Public Information Officer"
+      },
+      {
+        "name": "Ms. Seema Rani (Asstt. Programmer)",
+        "infoOfficer": "Nodal Officer"
+        }
```
[the remark this answers is not in the document]
Comment (dhruv-karan) [reply src/components/static/Rti/RtiData.json]: done

File: src/components/static/Rti/Rti.js [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+`;
+
+const P = styled.p`
+  font-weight: 600;
+  font-family: 'Open Sans', sans-serif;
+`;
+
+export default () => (
+  <Container>
+    <H2ResAuto>RTI(RIGHT TO INFORMATION ACT) </H2ResAuto>
```
[the remark this answers is not in the document]
Comment (dhruv-karan) [reply src/components/static/Rti/Rti.js]: done

File: src/components/static/Faculty/FacultyLines.js [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+      View
+    </A>
+    <div>
+      <Head> Faculty </Head>
+      {facility.map((item, i = 0) => <Name key={i++}> {item} </Name>)}
+    </div>
+  </Department>
+);
+
+module.exports = {
```
[the remark this answers is not in the document]
Comment (dhruv-karan) [reply src/components/static/Faculty/FacultyLines.js]: i need to make 2 compontents bcz they have totally different data sets

[the remark this answers is not in the document]
Comment (s-r-aman) [reply src/components/static/Faculty/FacultyLines.js]: And how does this solve the problem in which ES6 syntax cant solve it?

**Pull request merged**